### ISSUE TITLE
remove wwctl overlay build --host

### ISSUE THREAD

**Issue by anderbubble** [warewulf/warewulf#1419]
I suggest that the host overlay should only be used by `wwctl configure`. As such, we should remove `wwctl overlay build --host` and probably `wwctl overlay build --nodes` too, since that would be the only behavior at that point.

**Comment by mslacken** [comment]
Right. The `wwctl overlay build --host` was introduced at a time when code logic didn't allow to render single overlays, but just all at once. So there was need for this switch, so we can remove it right now.
